stateMatrix.m

```diff
@@ -23,6 +25,9 @@
 RightPWM = round((100-TaskParameters.GUI.RightPokeAttenPrcnt) * 2.55);
 LEDErrorRate = 0.1;
 
+LeftSoundPWM = round((100-TaskParameters.GUI.LeftSoundAttenPrcnt) * 2.55);
+RightSoundPWM = round((100-TaskParameters.GUI.RightSoundAttenPrcnt) * 2.55);
+
```

Comment:
while we have a light-intensity experiment type, but we also use the light as a cue for different things (e.g early-withdrawal and starting a new trial), so calculating the PWM for light makes sense regardless of the experiment time, but for sound here, it's only specific to auditory level experiment-types, therefore I suggest moving these three lines below [here](https://github.com/ioannapandi/Mouse2AFC/pull/2/files#diff-852d10613a3c2089f482d427811de95fR50)